ansible/roles/ocp4-workload-security-compliance-lab/tasks/deploy_gogs.yml

```diff
@@ -1,11 +1,12 @@
 ---
-## Deploy Gogs Git Server
+
```

Comment:
These should be refactored to use `k8s` module whenever possible. This can be a followup to change after this is initially merged.